src/components/TypesOfRecipeCards/RecipeCard.jsx

```diff
@@ -0,0 +1,88 @@
+import { PushPin } from "@phosphor-icons/react";
+
+//RecipeCard is now able to take in recipe and review, and render the correct information
+
+const RecipeCard = ({ pinned, recipe }) => {
+  const white = "#d4d4d4";
+  const size = 18;
+  const orginalRecipe = recipe.recipeId ? recipe.recipeId : recipe;
+
+  const formattedReview =
+    recipe.userReview?.slice(0, 50) +
+    (recipe.userReview?.length > 50 ? "..." : "");
```

Comment:
You could just used `text-overflow` css
https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow